stubs/setuptools/setuptools/__init__.pyi

```diff
@@ -45,6 +46,14 @@ __all__ = [
 
 __version__: str
 
+@type_check_only
+class _BuildInfo(TypedDict):
+    sources: list[str] | tuple[str]
```

Comment:
Are you sure this isn't `tuple[str, ...]`? In other words, is this really always a one-tuple? (A few more times below.)

Reply:
You're right, I also glanced over that...
```suggestion
    sources: list[str] | tuple[str, ...]
```

Reply:
The one tuple for `macros` is correct, but the others were a mistake: https://github.com/python/typeshed/pull/12791/commits/2bb4ed4fd83d0b838393db15a1aa8753de02e112